### PR TITLE
many2many in filters

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/ralph/admin/filters.py [hunk 1]
```diff
@@ -277,21 +277,28 @@ class RelatedAutocompleteFieldListFilter(RelatedFieldListFilter):
     """Filter for Foregin key field."""
 
     template = "admin/filters/related_filter.html"
-    empty_value = '##@_empty_@##'
+    empty_value = 0
```
Comment (mkurek) [inline src/ralph/admin/filters.py]: move this to some "global" constant in some module and reuse it in others

Comment (ar4s) [inline src/ralph/admin/filters.py]: :+1:

**Comment by ghost** [comment]
@mkurek @ar4s done

**Comment by ar4s** [comment]
:ok:

**Pull request closed without merging**